### PR TITLE
Enables 515 tests

### PR DESCRIPTION
**Pull request by AnturK**
Random gc failures seem to be gone and savefile compatibility working in 515.1596. So let's try enabling this.

### CODE REVIEW

File: .github/alternate_byond_versions.txt [hunk 1]
```diff
@@ -7,4 +7,4 @@
 # 500.1337: runtimestation
 
 # Commented out until 515 is fully ready
```
Comment (JohnFulpWillard) [inline .github/alternate_byond_versions.txt:9]:
```suggestion
```

**Comment by san7890** [comment]
image

Thank you Auxlua, very cool!

**Comment by Mothblocks** [comment]
That's not surprising, auxtools needs fixing

**Comment by Mothblocks** [comment]
I'll start making this required after a week or so when most people are up to date, and I turned on the button to let us update branches easily anyway

**Pull request merged**